packages/pods-core/src/v2/podConfig/GoogleDocsConfig.ts

```diff
@@ -71,7 +73,7 @@ export function createRunnableGoogleDocsV2PodConfigSchema(): JSONSchemaType<Runn
       exportScope: {
         type: "string",
       },
-      description: {
+      parentFolderId: {
```

Comment:
maybe `parentFolder` instead of `parentFolderId`?  The value here is human readable right? Like "notes", not some GUID?

Reply:
The value is GUID. The only readable value is `root` which means at the root of drive.

Reply:
Ah ok - in that case let's leave it as `parentFolderId`